spec/models/searchs/vacancy_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+describe Searchs::Vacancy, type: :search do
+  describe '.list' do
+    context 'without parameter' do
+      it 'returns all vacancies' do
+        vacancies = create_list(:vacancy, 5)
+
+        expect(Searchs::Vacancy.list).to eq vacancies.reverse
+      end
+    end
+
+    context 'with parameter' do
+      it 'returns selected results' do
+        vacancy_one = create(:vacancy, job_title: 'java programador')
+        create(:vacancy, job_title: 'programador ruby')
+
+        expect(Searchs::Vacancy.list('programadores java')).to eq([vacancy_one])
+      end
+
+      it 'returns results that contains search param' do
+        vacancy_one = create(:vacancy, job_title: 'programador ruby')
+        create(:vacancy, job_title: 'programador java')
+
+        expect(Searchs::Vacancy.list('programador ruby')).to eq([vacancy_one])
+      end
+    end
```

Comment:
Yeah.. Think this is duplicated =). The old one since it was made with english dictionary, it was possibly to find 'dev' for 'developer'. Now we are fixed on portuguese one. I will remove this one.